modules/platform/nuxeo-platform-relations/nuxeo-platform-relations-core/src/test/java/org/nuxeo/ecm/platform/relations/TestRelationService.java

```diff
@@ -158,6 +166,22 @@ public void testGetResourceUnexistent() {
         assertNull(resource);
     }
 
+    @Test
+    public void testGetResourceUnexistentInvalid() throws IOException {
+        URL url = Thread.currentThread().getContextClassLoader().getResource("nxrelations-test-invalid.xml");
+        Framework.getRuntime().getContext().deploy(url);
+        Framework.getRuntime().getComponentManager().unstash();
```

Comment:
Ah indeed I tried that but undid the change because the ClassNotFound is not correctly caught and prevents the test from going through: I'll cleanup the hotDeployer addition and add a note to https://jira.nuxeo.com/browse/NXP-30092 about it.